src/entrypoint.sh

```diff
@@ -12,8 +12,8 @@ source "${SCRIPT_DIR}/sync_common.sh"
 # Precheks
 ##########################################
 
-if [[ -z "${ORG_GITHUB_TOKEN}" ]]; then
```

Comment:
Just wanted to make it clear with the naming that the token is related to the source repo